### PR TITLE
Fix Boar Gift bug 

### PR DESCRIPTION
**Pull request by Double2Proj**
Fix boar gift buttons only being placed in the top row.
Resubmitted to dev branch at Weslay's request.

### CODE REVIEW

**Comment by WeslayCodes** [comment]
test

**Comment by WeslayCodes** [comment]
i am writing this comment as a test

**Review by WeslayCodes — COMMENTED** [review]
test

**Review by WeslayCodes — DISMISSED** [review]
test2

File: src/main/java/dev/boarbot/interactives/gift/BoarGiftInteractive.java [hunk 1]
```diff
@@ -579,7 +579,7 @@ public void stop(StopType type) {
     public ActionRow[] getCurComponents() {
         List<ActionRow> rows = new ArrayList<>();
         int SIDE_LENGTH = 3;
-        int claimIndex = (int) (Math.random() * SIDE_LENGTH);
```
Comment (WeslayCodes) [inline src/main/java/dev/boarbot/interactives/gift/BoarGiftInteractive.java:582]: test again

**Review by WeslayCodes — COMMENTED** [review]
another test

File: src/main/java/dev/boarbot/interactives/gift/BoarGiftInteractive.java
Comment (WeslayCodes) [inline src/main/java/dev/boarbot/interactives/gift/BoarGiftInteractive.java:1]: a

**Comment by WeslayCodes** [comment]
test?

**Review by WeslayCodes — CHANGES_REQUESTED** [review]
Do you think you could expand the scope to fix #6 completely in this PR? See [components.json](https://github.com/BoarBotDevs/BoarBot/blob/d50def3fc718ec3cc3be3d2c6540565cbdb38d32/src/main/resources/config/discord/components.json) and [pow_prompts.json](https://github.com/BoarBotDevs/BoarBot/blob/d50def3fc718ec3cc3be3d2c6540565cbdb38d32/src/main/resources/config/game/pow_prompts.json). Should be as simple as substituting the empty character with an emoji.

**Comment by Double2Proj** [comment]
The gift one just seems to be replacing a string - but aren't quick click buttons pulling from the empty string in string_config.java? I think I need to change line 328 in PowerupEventInteractive.java, though I'm not super familiar with how the bot fetches strings. I suppose it could be just be hardcoded and I change it directly in PowerupEventInteractive to an emoji?

**Comment by WeslayCodes** [comment]
I guess that one is a bit more complicated. I'll make it a separate issue in a bit. Will have different requirements.

File: src/main/resources/config/discord/components.json [hunk 2]
```diff
@@ -589,4 +589,4 @@
       "style": 4
     }
   }
-}
```
Comment (WeslayCodes) [inline src/main/resources/config/discord/components.json]: This is something I've configured in my project, should probably do the same for yours (if you care): https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings

Comment (Double2Proj) [reply src/main/resources/config/discord/components.json]: Ah, I see. I've configured it now.

**Review by WeslayCodes — DISMISSED** [review]

**Comment by WeslayCodes** [comment]
Seems your latest commit wasn't signed, here's a guide for how to set that up: https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits. And here's how to do that for previous commits: https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits

It's a lot, but it's used to verify that you are who you say you are.

**Review by WeslayCodes — APPROVED** [review]

**Pull request closed without merging**